mmv1/products/gkehub2/Feature.yaml

```diff
@@ -93,6 +93,11 @@ examples:
     skip_test: true
     primary_resource_name: 'fmt.Sprintf("configmanagement")'
     primary_resource_id: 'feature'
+  - !ruby/object:Provider::Terraform::Examples
+    name: 'enable_fleet_default_member_config_policycontroller'
+    skip_test: true
```

Comment:
@tonyzhc Hi, what's the main reason for skipping these? Were you able to test locally?

Reply:
The test and the API dependencies don't work well with VCR tests. We instead covered them in the custom acceptance tests and added the skip VCR options. I have added more examples to serve the documentation purpose.